### DISCUSSION TITLE
Deno support for server side?

### DISCUSSION THREAD

**Discussion by Nukiloco** [trpc/trpc#1706] [Q&A]
Is there going to be Deno support for the server side of this? Thanks!

**Comment by sachinraja** [comment]
See #1597

**Reply by KATT** [reply]
Additionally, I've never done a deno module, but should be pretty straight-forward once that is done

**Reply by omar2205** [reply]
> Additionally, I've never done a deno module, but should be pretty straight-forward once that is done

That issue is closed but still, IDK how to use it with Deno.

**Reply by sachinraja** [reply]
See https://github.com/sachinraja/trpc-ultra. It's not great because Deno doesn't have any real support for peerDependencies.

**Comment by samifouad** [comment]
deno support works as expected. it may not be obvious from the docs, but as @KATT + @sachinraja indicated, it works.

here's a hello world example repo I published: https://github.com/samifouad/trpc-deno-hello-world

hope it can give you a basic understanding so you can build up from there. cheers.

*side note: my example uses an Express port to Deno called opine. [Deno v1.25](https://deno.com/blog/v1.25) is going to be shipping with experimental npm support. this means that technically you could try using Express directly and a bunch of other npm packages that can help get you setup with tRPC a little easier*
